wled00/wled00.ino

```diff
@@ -507,6 +505,7 @@ void setup() {
 
 //main program loop
 void loop() {
+  handleIR();          //2nd call to function needed for ESP32 to return valid results -- should be good for ESP8266, too
```

Comment:
Especially on ESP32 the IR-receiver did not work well and a lot of false values were received.
Adding this line does "double" the IR receiving time and did improve the quality so that no more wrong values were received.